collect_app/src/main/java/org/odk/collect/android/activities/GeoPointActivity.java

```diff
@@ -71,6 +73,12 @@
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+
+        if (!checkIfLocationPermissionsGranted(this)) {
```

Comment:
Is there any path to get here? From what I can see, the button to launch these activities won't launch them without the permission grant.

Reply:
I think if the user somehow manages to leave this screen and go to the `Settings` and disable the location permission then it would cause a crash when the activity is reopened so this check ensure that doesn't happen. I could double check to be sure and to see if this would belong in `onResume` :)

Reply:
Tricky! That makes sense @jd-alexander. @mmarciniak90 that would be a great case to verify.